web/yo/app/scripts/services/databaseconnector.js

```diff
@@ -45,28 +45,6 @@ angular.module('oncokbApp')
             var testing = OncoKB.config.testing || false;
             var inProduction = OncoKB.config.production || false;
 
-
-            function getAllDrugs() {
-                var deferred = $q.defer();
-                Drugs.getAllDrugs()
-                    .then(function(data) {
-                        deferred.resolve(data.data);
-                    }, function(error) {
-                        var subject = 'getAllDrugs Error';
-                        var content = 'The system error returned is ' + JSON.stringify(error);
-                        sendEmail({sendTo: 'dev.oncokb@gmail.com', subject: subject, content: content},
-                            function(result) {
-                                console.log('sent getAllDrugs Error to oncokb dev account');
-                            },
-                            function(error) {
-                                console.log('fail to send getAllDrugs Error to oncokb dev account', error);
-                            }
-                        );
-                        deferred.reject(error);
-                    });
-                return deferred.promise;
-            }
-
             function searchDrugs(keyword) {
```

Comment:
Why do you remove getAllDrugs()? Is it not be used any more?

Reply:
Firebase realtime database is the source now. We won't use API to get all drugs.